addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -67,6 +68,7 @@ class AttachmentViewer extends Component {
     mounted() {
         this.el.focus();
         this._handleImageLoad();
+        this._handlePdfButtons();
```

Comment:
You would want to do that also in `patched`, the element that you want to target is not guaranteed to be already there at mounted.